doc/contributing/test.rst

```diff
@@ -56,7 +56,7 @@ Create test databases:
 
 Set the permissions::
 
-    paster datastore set-permissions -c test-core.ini | sudo -u postgres psql
+    paster datastore set-permissions -c |virtualenv|/src/ckan/test-core.ini | sudo -u postgres psql
```

Comment:
In many cases, CKAN instance is configured in quite a unique way, so the assumption that `|virtualenv|/src/ckan/test-core.ini` exists won't be truthy all the time. In addition, `paster datastore`, without `--plugin=...` won't work from every directory, so I suggest to leave this place as it was before

Reply:
Yep, making it clear, that command have to be executed from particuar directory should be ok